components/inputs/input-date-range.js

```diff
@@ -43,7 +43,7 @@ class InputDateRange extends InteractiveMixin(FocusMixin(SkeletonMixin(FormEleme
 			 */
 			autoShiftDates: { attribute: 'auto-shift-dates', reflect: true, type: Boolean },
 			/**
-			 * Hides the start and end labels visually
+			 * ACCESSIBILITY: Hides the start and end labels visually
```

Comment:
Curious thoughts on this - for screen reader users showing/hiding these "labels hidden" properties wouldn't necessarily have an impact so I could see the argument that we wouldn't want to add the prefix to those ones

Reply:
Yeah, agreed — I don't think we need the `ACCESSIBILITY:` prefix on this one